pkg/accelerator/accelerator.go

```diff
@@ -0,0 +1,84 @@
+//
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
+
+package accelerator
+
+import (
+	"fmt"
+
+	"github.com/jaypipes/ghw/pkg/context"
+	"github.com/jaypipes/ghw/pkg/marshal"
+	"github.com/jaypipes/ghw/pkg/option"
+	"github.com/jaypipes/ghw/pkg/pci"
+)
+
+type AcceleratorCard struct {
```

Comment:
Instead of calling it `AcceleratorCard`, what do you think about `AcceleratorDevice`? Not all accelerators are necessarily "cards", after all, and "device" is a bit more generic...

Reply:
good idea, _device_ is much more generic and appropriate than _card_